src/sdk/pynni/nni/compression/tensorflow/default_layers.py

```diff
@@ -1,8 +1,21 @@
-op_weight_index = {
-    'Conv2D': None,
-    'Conv3D': None,
-    'DepthwiseConv2dNative': None,
+from tensorflow import keras
 
-    'Mul': None,
-    'MatMul': None,
+supported_layers = {
+    keras.layers.Dense: ('Dense', 0),
+    keras.layers.Conv1D: ('Conv1D', 0),
+    keras.layers.Conv2D: ('Conv2D', 0),
 }
+
+default_layers = ['Dense', 'Conv2D']
```

Comment:
what is the difference between `supported_layers` and `default_layers`?

Reply:
Thanks, updated, please check again.